src/StripeTests/Entities/StripeApplicationFeeTest.cs

```diff
@@ -0,0 +1,24 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeApplicationFeeTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            string json = GetFixture("/v1/application_fees/fee_123");
+            var applicationFee = Mapper<StripeApplicationFee>.MapFromJson(json);
+            Assert.NotNull(applicationFee);
+            Assert.IsType<StripeApplicationFee>(applicationFee);
+            Assert.NotNull(applicationFee.Id);
+            Assert.Equal("application_fee", applicationFee.Object);
+        }
+    }
```

Comment:
This class has expandable attributes. Could you add a test like [this one](https://github.com/stripe/stripe-java/blob/master/src/test/java/com/stripe/model/ApplicationFeeTest.java#L28) in stripe-java?

`GetFixture()` already supports a second `string[]` arguments with the list of attributes you want to expand.

Reply:
Nice catch, added a test for that!